includes/gateways/stripe/class-give-stripe.php

```diff
@@ -68,6 +68,41 @@ public function includes() {
 
 			// Bailout, if any of the Stripe gateway is not active.
 			if ( ! give_stripe_is_any_payment_method_active() ) {
+
+				// If `get_plugin_data` fn not exists then include the file.
+				if ( ! function_exists( 'get_plugin_data' ) ) {
+					require_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+				}
+
+				// Hardcoded recurring plugin basename to show notice even when recurring addon is deactivated.
+				$recurring_plugin_basename = 'give-recurring/give-recurring.php';
+				$recurring_plugin_data     = get_plugin_data( WP_CONTENT_DIR . '/plugins/' . $recurring_plugin_basename );
```

Comment:
@mehul0810 you should use https://developer.wordpress.org/reference/functions/plugin_dir_path/ or similar in case this isn't the standard path.